src/routes/flavor.js

```diff
@@ -308,4 +315,69 @@ router.delete(
   })
 );
 
+/**
+ * GET Flavor Notes
+ * @query offset int
+ * @query limit int
+ */
+router.get(
+  '/:flavorId/notes',
+  authenticate(),
+  ensurePermission('flavors', 'read'),
+  [
+    param('flavorId').isNumeric().isInt({ min: 1 }).toInt(),
+    query('offset').optional().isNumeric().toInt(),
```

Comment:
These should probably be `isInt({ min: 1 })` as well, right? No floating point offsets or limits.